hieradata/node/atarchiver.cp.lsst.org.yaml

```diff
@@ -76,6 +76,25 @@ nfs::nfs_exports_global:
       139.229.160.0/24(ro,nohide,insecure,no_subtree_check,async,root_squash)
       139.229.165.0/24(ro,nohide,insecure,no_subtree_check,async,root_squash)
       139.229.175.128/25(ro,nohide,insecure,no_subtree_check,async,root_squash)
+  /data/repo:
+    clients: >-
+      %{facts.networking.ip}/32(ro,nohide,insecure,no_subtree_check,async,root_squash)
+      139.229.146.0/24(rw,nohide,insecure,no_subtree_check,async,root_squash)
```

Comment:
Will these mandate NFS v4?  That is necessary for locking (or else we have to ensure a bunch of other things are correct for NFS v3).

Reply:
I believe it does, the puppet package at least is for nfs4